src/components/Endpoint.tsx

```diff
@@ -99,21 +109,44 @@ const Endpoint = (props) => {
     setEndpointList({ DATA: [], path: null });
     setLoading(true);
     try {
-      let fullPath = query.get('full-path');
-      let url = `operation_ls?endpoint=${endpointID}`;
+      var fullPath = query.get('full-path');
+      var url = `operation_ls?endpoint=${endpointID}`;
       if (fullPath) {
         url = `${url}&path=${fullPath}`;
       }
       const listItems = await requestAPI<any>(url);
       setEndpointList(listItems);
     } catch (error) {
+      setAPIError({ ...error, ...{ global: true } });
       let error_response = await error.response.json();
-      /* Note: This probably isn't a great UX to simply pop up a login page, but it
-      does demonstrate the base functionality for picking endpoints */
+
+      /* 
+        Note: This probably isn't a great UX to simply pop up a login page, but it
+        does demonstrate the base functionality for picking endpoints 
+      */
```

Comment:
We should put in a ticket for updating this behavior.

Reply:
Sounds good. I'll need to brainstorm with you how you might want to change the behavior.

Reply:
Added a ticket for this: https://app.shortcut.com/globus/story/16122